src/components/MessagesList/MessagesList.vue

```diff
@@ -643,8 +642,7 @@ export default {
 		},
 
 		async handleStartGettingMessagesPreconditions() {
-			if (this.token && this.isParticipant && !this.isInLobby) {
-
+			if (this.token && !this.isInLobby) {
```

Comment:
I would avoid to skip that part, that could lead to some unexpected things.
Though as we knowing some data from conversation, we could do like this (conversation contains the same params as partticipantIdentifier for that conversation, and already known to us):

```js
isParticipant() {
	if (!this.conversation) {
		return false
	}

	return !!this.$store.getters.findParticipant(this.token, this.conversation)
},
```

Reply:
Perfect 😀